src/main/java/org/opensearch/ad/rest/RestPreviewAnomalyDetectorAction.java

```diff
@@ -11,8 +11,9 @@
 
 package org.opensearch.ad.rest;
 
-import static org.opensearch.ad.util.RestHandlerUtils.PREVIEW;
+import static org.opensearch.ad.util.RestHandlerUtils.*;
```

Comment:
Don't use star imports

Reply:
the pain of intellij and its extra smartness :D. I will fix this.